e2e_tests/postgres/producer/producer.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/segmentio/kafka-go"
+)
+
+// DebeziumMessage represents the structure of messages in the test data files
+type DebeziumMessage struct {
+	Schema  json.RawMessage `json:"schema"`
+	Payload json.RawMessage `json:"payload"`
+}
+
+// TopicMapping defines which file goes to which topic
+type TopicMapping struct {
+	FilePath string
+	Topic    string
+}
+
+func main() {
+	ctx := context.Background()
```

Comment:
Can this read from the Transfer config file so we don't need to maintain 2 SoTs for Kafka conn?